src/api/v1/jsonwebtoken.js

```diff
@@ -0,0 +1,8 @@
+import jwt from "jsonwebtoken";
+import dotenv from "dotenv";
+dotenv.config();
+
+export const generateToken = (userId) => {
+	const secretKey = process.env.SECRET_KEY;
```

Comment:
make the SECRET_KEY a constant and put in the constants.js file, so you can import it here instead. You shouldn't throw dotenv into a whole lot of files. env variables are to be collected in one place and then exported for all other modules to use. This is the convention for any configuration type of logic. Instead of calling the dotenv.config() function in multiple files, you do it in one place and export the needed variables.